docs/notebooks/beta_regression.ipynb

```diff
@@ -0,0 +1,1535 @@
+{
```

Comment:
<p>I think something like this is simpler and shorter.</p><p> </p><p>def mukappa_to_alphabeta(mu, kappa):</p><p>&nbsp;&nbsp;&nbsp;alpha = mu * kappa</p><p>&nbsp;&nbsp;&nbsp;beta = (1 - mu) * kappa</p><p>&nbsp;&nbsp;&nbsp;alpha_mean = alpha.mean(["chain", "draw"]).values</p><p>&nbsp;&nbsp;&nbsp;alpha_hdi = az.hdi(alpha, hdi_prob=.95)["x"].values</p><p>&nbsp;&nbsp;&nbsp;beta_mean = beta.mean(["chain", "draw"]).values</p><p>&nbsp;&nbsp;&nbsp;beta_hdi = az.hdi(beta, hdi_prob=.95)["x"].values</p><p>&nbsp;&nbsp;&nbsp;return alpha_mean, alpha_hdi, beta_mean, beta_hdi</p><p> </p><p>alpha, alpha_hdi, beta, beta_hdi = mukappa_to_alphabeta(</p><p>&nbsp;&nbsp;&nbsp;expit(fitted.posterior["Intercept"]),</p><p>&nbsp;&nbsp;&nbsp;fitted.posterior["probabilities_kappa"])</p><p> </p><p>print(f"Alpha - mean: {np.round(alpha)}, 95% HDI: {np.round(alpha_lower)} - {np.round(alpha_upper)}")</p><p>print(f"Beta - mean: {np.round(beta)}, 95% HDI: {np.round(beta_lower)} - {np.round(beta_upper)}")</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/442/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Good thought. Implemented.

Reply:
I think you miss this change.

Reply:
<p>This can be deleted now</p>